src/Alpha_complex/utilities/alpha_complex_persistence.cpp

```diff
@@ -97,12 +109,13 @@ int main(int argc, char **argv) {
   std::string output_file_diag;
   bool exact_version = false;
   bool fast_version = false;
+  bool output_squared_root_values = false;
```

Comment:
`output_squared_root_values`
There are also several below, I will stop pointing them out.

Reply:
Modified by [44a6747](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/44a6747c1c2ffd0d80db14bfd65d64cd9f0e6e88), [4227322](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/4227322a044ffd87c2a0d3743d8bd9a6f1d66bf1) and [404cb39](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/404cb3951e1c9f6283a2ccc3c39ce340b65f9ddd)